projects/fal/tests/conftest.py

```diff
@@ -9,4 +9,4 @@
 
 @pytest.fixture(scope="function")
 def isolated_client():
-    return partial(function, machine_type="M", keep_alive=0)
+    return partial(function, machine_type="XS", keep_alive=0)
```

Comment:
Also switched to XS machines and it made it much easier and faster to aquire workers. Ideally in the future it would be nice to test everything we can locally, there is really no reason to waste resources.